### PR TITLE
Archive garden-windows-ci repo

### PR DESCRIPTION
**Pull request by ebroberson**
Recently we had some issues with the windows-rootfs pipeline failing because of its references to both [garden-windows-ci](https://github.com/cloudfoundry/garden-windows-ci) and the [WG CI repo](https://github.com/cloudfoundry/wg-app-platform-runtime-ci). The conflict arose because we have some opsfiles that have the same name, but different functionality. We were able to fix this by removing the bad opsfile, but it would be better if we had one source of truth for our CI pipelines.

Since `garden-windows-ci` is no longer referenced by our pipelines and the relevant files are now in the WG CI repo, we should archive `garden-windows-ci`.

### CODE REVIEW

**Review by ameowlia — APPROVED** [review]

**Comment by ameowlia** [comment]
This is for my working group and I approve. I can't remember if I need to wait for full TOC approval.

**Review by beyhan — APPROVED** [review]

**Comment by beyhan** [comment]
Usually, I take this into the TOC meeting but if we get another approval by @rkoster, @stephanme or @ChrisMcGowan we don't need to wait.

**Comment by stephanme** [comment]
You should remove the project from https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-platform.md (repositories of area Garden Containers) as well.

**Review by rkoster — APPROVED** [review]

**Comment by beyhan** [comment]
@stephanme thanks for the good catch. The request has been implemented and we have enough approvals I will merge to sped up things.

**Pull request merged**
